src/Diagnostics/graph_diagnostic.jl

```diff
@@ -11,45 +40,396 @@ Usage:
 end
 
 function start(args::Vector{String})
-    data = load(args[1])
+    #data = load(args[1])
+    
+    #
+    # USER INPUTS:
+    #
+    user_specified_timestart = 0
+    user_specified_timeend   = -12329.963523831071 # set it to -1 if you want the plotter to detect and show the last time step data only
+    time_average = "y"
+    isimex = "y"
+    dh=40
+    dv=20
+
+    #
+    # List the directories containing the JLD2 files to post-process:
+    #
+    gcloud_VM = ["yt-DYCOMS-MULTI-RATE-NEW-MOIST-THERMO"] #EDDY
+        
+    for gcloud in gcloud_VM
+
+        
+        SGS = "Smago"
+        
+        ode_str = "imex"
+        
+        #data = load(string(clima_path, "/output/EDDY/", gcloud, "/IMPL/diagnostics-2020-01-16T22:11:26.296.jld2")) #Dsub=0 component-wise Geostrophic forcing
+
+        data = load(string(clima_path, "/output/EDDY/", gcloud, "/IMPL/diagnostics-2020-01-17T07:44:12.648.jld2")) # D=-3.7 
+        
+        
+        
+    info_str = string(dh,"X", dv, "-SGS-", ode_str)
+        
+    out_vars = ["ht_sgs",
+                "qt_sgs",
+                "h_m",
+                "h_t",
+                "vert_eddy_qt_flx",
+                "q_tot",
+                "q_liq",
+                "wvariance",
+                "wskew",
+                "thd",
+                "thv",
+                "thl",
+                "w",
+                "uvariance",
+                "vvariance",
+                "vert_eddy_thv_flx",
+                "u",
+                "v"]
+               
+    
+    zvertical = 1500
+    #
+    # END USER INPUTS:
+    #
+
+
+    #
+    # Stevens et al. 2005 measurements:
+    #
+    qt_stevens  = readdlm(string(clima_path, "/output/Stevens2005Data/experimental_qt_stevens2005.csv"), ',', Float64)
+    ql_stevens  = readdlm(string(clima_path, "/output/Stevens2005Data/experimental_ql_stevens2005.csv"), ',', Float64)
+    thl_stevens = readdlm(string(clima_path, "/output/Stevens2005Data/experimental_thetal_stevens2005.csv"), ',', Float64)
+    tkelower_stevens = readdlm(string(clima_path, "/output/Stevens2005Data/lower_limit_tke_time_stevens2005.csv"), ',', Float64)
+    tkeupper_stevens = readdlm(string(clima_path, "/output/Stevens2005Data/upper_limit_tke_time_stevens2005.csv"), ',', Float64)
```

Comment:
@simonbyrne  these are data files that contain measurements for this benchmark. Where should they be stored?

Reply:
I can configure `DataDeps` for these file, if that would be helpful.

Reply:
> I can configure `DataDeps` for these file, if that would be helpful.

@charleskawczynski  sure, thanks.

You can find the data in this dropbox directory: https://www.dropbox.com/sh/jwmrgs5b2jjtnnn/AABXOYFbF9obNfbnKqVMZUTla?dl=0

Reply:
These are DYCOMS specific (`104-108`) - do we want to include this in the general master `graph_diagnostics.jl` ?

Reply:
We discussed creating a `VizCLIMA` repo (cc: @glwagner) to handle plotting diagnostics. Maybe it would be a good idea to start this repo?

Reply:
Sounds like a good plan. I've made quite a few modifications to `graph_diagnostics` to make it easier to use and to create publication quality plots. 
I will add it to the repo once it's been created.